magma/backend/mlir/hardware_module.py

```diff
@@ -186,7 +186,7 @@ def __init__(self, graph: Graph, ctx):
         self._ctx = ctx
         self._visited = set()
 
-    @functools.lru_cache()
+    #@functools.lru_cache()
```

Comment:
similar to the other comment, we unfortunately can't blindly use cached constants, since some might be inside of compile guards (we *might* be able to check for this, but I think this starts to get pretty gross).